packaging/datadog-agent/source/install_agent.sh

```diff
@@ -144,7 +144,11 @@ else
 fi
 
 printf "\033[34m* Starting the Agent...\n\033[0m\n"
-$sudo_cmd /etc/init.d/datadog-agent restart
+if [ $OS = "Debian" ]; then
+    $sudo_cmd service datadog-agent restart
+else
+    $sudo_cmd /etc/init.d/datadog-agent restart
+fi
```

Comment:
Same questions as the other PR actuatlly. And here why do we use `service` and not `invoke-rc.d` then?